projects/transfer-journey/src/lib/views/make-transfer.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { MakeTransferJourneyConfigService } from "../make-transfer-journey-config.service";
+import { Account } from "../model/Account";
+
+@Component({
+  templateUrl: 'make-transfer.component.html'
+})
+export class MakeTransferComponent {
+  account: Account = {
```

Comment:
As for me, better to save this to a variable, something like `accountMock`

Reply:
done